tests/test-core/testTests/js/IoCTestingTests.js

```diff
@@ -307,14 +317,20 @@ fluid.defaults("fluid.tests.fluid5559Tree", {
                     name: "FLUID-5559 Double firing of onTestCaseStart",
                     tests: [{
                         name: "FLUID-5559 sequence",
-                        expect: 2,
-                        sequence: [ {
+                        expect: 3,
+                        sequence: [{
                             // Must use IoCSS here - see discussion on FLUID-4929 - must avoid triggering construction
-                            event: "{fluid5559Tree targetTree}.events.onCreate",
-                            listener: "fluid.tests.onTestCaseStart.assertOnce"
+                            event: "{fluid5559Tree targetTree}.events.hearIt",
+                            listener: "fluid.tests.onTestCaseStart.assertValue"
                         }, {
-                            func: "fluid.tests.onTestCaseStart.assertOnce",
+                            func: "fluid.tests.onTestCaseStart.assertValue",
                             args: "{targetTree}"
+                        }, { // Try firing the event again in order to test FLUID-5633
+                            func: "{fluid5559Tree}.targetTree.events.hearIt.fire",
+                            args: 5
+                        }, {
+                            event: "{fluid5559Tree targetTree}.events.hearIt",
+                            listener: "fluid.tests.onTestCaseStart.assertValue"
```

Comment:
Shall the value be verified to ensure that it receives the argument from the second firing of the `hearIt` event instead of the residue of the first registered listener on this event?